apps/pkey.c

```diff
@@ -175,15 +175,19 @@ int pkey_main(int argc, char **argv)
     if (argc != 0)
         goto opthelp;
 
-    if (noout && pubout)
```

Comment:
Why remove this?  The user has asked for no output and pubout.  This should be an error.  If anything I'd make it fatal rather than a warning.

Reply:
`-noout` doesn't mean no output at all.  `-noout` means no *encoded* (i.e. PEM or DER) output.
For example, you can combine `-noout` with `-text`.

There's really no reason why `-pubout` should be reserved for PEM / DER output only.

Reply:
I agree that the option name `-noout` sucks.  However, we have lived with it for 20+ years...